src/transfer/transfer.interface.ts

```diff
@@ -0,0 +1,11 @@
+export interface ITransfer {
+    _id?: string;
+    fileID: string;
+    userID: string;
```

Comment:
I'm a bit confused. Is the `userID` the user how created the trasfer or the user the file is transferred to?

Reply:
userID is the id of the user who created the transfer.
I think you are right and the user the file is transferred to is relevant too.

Reply:
the userID is the id of the sharer. when  I create a request to dropbox i pass the id of the transfer, and I can get the users that the file was transferred to.

Reply:
So what is the purpose of `HasTransfer`?
To check if a user shared a file, or to check if a file was transferred for a user?